packages/@okta/vuepress-site/docs/reference/api/system-log/index.md

```diff
@@ -280,38 +280,38 @@ In the Okta event data object, the UserAgent object provides specifications abou
 
 | Property    | Description                                                                                                          | DataType                                  | Nullable |
 | ----------- | ----------------------------------------------------------------------------------                                   | -------------------------------------     | -------- |
 | geolocation | Contains the geolocation coordinates (latitude, longitude)                                                           | [Geolocation object](#geolocation-object) | TRUE     |
-| city        | The city encompassing the area containing the geolocation coordinates, if available (e.g. Seattle, San Francisco)    | String                                    | TRUE     |
-| state       | Full name of the state/province encompassing the area containing the geolocation coordinates (e.g. Montana, Incheon) | String                                    | TRUE     |
-| country     | Full name of the country encompassing the area containing the geolocation coordinates (e.g. France, Uganda)          | String                                    | TRUE     |
-| postalCode  | Postal code of the area encompassing the geolocation coordinates                                                     | String                                    | TRUE     |
+| city        | The city that encompasses the area that contains the geolocation coordinates, if available (for example, Seattle, San Francisco)    | String                                    | TRUE     |
+| state       | Full name of the state/province that encompasses the area that contains the geolocation coordinates (for example, Montana, Incheon) | String                                    | TRUE     |
+| country     | Full name of the country that encompasses the area that contains the geolocation coordinates (for example, France, Uganda)          | String                                    | TRUE     |
+| postalCode  | Postal code of the area that encompasses the geolocation coordinates                                                     | String                                    | TRUE     |
 
 ### Geolocation object
 
-Latitude and longitude of the geolocation where an action was performed, formatted according to the [ISO-6709](https://en.wikipedia.org/wiki/ISO_6709) standard.
+The latitude and longitude of the geolocation where an action was performed that is formatted according to the [ISO-6709](https://en.wikipedia.org/wiki/ISO_6709) standard
```

Comment:
Just found this a little clunky. Maybe break into two sentences: "The latitude and longitude of the geolocation where an action was performed. The object is formatted according to ...."